src/read_write/mod.rs

```diff
@@ -38,3 +38,5 @@ pub use self::raw::{RawNodeReader, RawNodeWriter};
 
 mod s2;
 pub use self::s2::S2Splitter;
+
+pub const BUFFER_CAPACITY_1MB: usize = 1024 * 1024;
```

Comment:
How about just having a `pub const MB: u64 = 1024 * 1024;` in `src`, or, if it should be local to read/write, remove the `_1MB` suffix?

Reply:
i changed it into pub const MB: usize = 1024 * 1024; in src

Reply:
.. usize because capacity mostly requires usize type